server/prefect/requirements.txt

```diff
@@ -44,3 +44,4 @@ websocket-client==0.58.0
 websockets==8.1
 zict==2.0.0
 zipp==3.4.1
+pytest==7.1.2
```

Comment:
Please insert pytest in alphabetical order

Reply:
ok, done

Reply:
added a line to run prefect pytest in /.github/workflows/build_test_backend.yml as per your note regarding continuous integration